### PR TITLE
Journal chap 1-3

### PR DESCRIPTION
**Pull request by juliegugge**
@jaredh159 @Henderjay

### CODE REVIEW

**Comment by juliegugge** [comment]
Morning you guys.... I hope you have a good day today... Just wanted to tell you that at one point in fixing the typos I noticed that several lines were "gone" from the modern version -- in chapter one, the long paragraph that starts with When we were thus in our deep fears: the sentences : "and great dread and trembling fell upon us.... etc"... I don't know if I bumped something by accident or what... I went to the original version and copy/pasted the missing section and then corrected any typos, and I think I took care of it but I thought I'd tell you in case the PR version shows something missing... Also, there are a few letters that need formatting -- they don't match the rest of the letters... I tried to insert [.salutation] etc to see if it would change it and it didn't look like it did anything so I put it back.... True confessions... sigh.... I was also just a tad shy to insert these comments to you in the text, hence the comment here.... thanks

File: journal-letters/modernized/03-chapter-1.adoc [hunk 1]
```diff
@@ -129,15 +129,15 @@ occasioned wrath,) and would sprinkle our hearts from an evil conscience,
 and wash us with pure water; that we might draw near with a true heart,
 in full assurance of faith, as the Christians of old did. Heb. 10:22.
 
-But, alas! we had not boldness,--for the living faith was lacking;
+But, alas--we had not boldness,--for the living faith was lacking;
```
Comment (Henderjay) [inline journal-letters/modernized/03-chapter-1.adoc]: I think we want to leave the exclamation point in, and _then_ add the double dash.  Like: `But, alas!--we had`

File: journal-letters/modernized/03-chapter-1.adoc [hunk 2]
```diff
@@ -222,17 +222,17 @@ who in the joy of their souls broke forth in praises unto the Lord;
...
-and to meet oftener together than before,-- our hearts
+and to meet together more often than before,-- our hearts
 being affected with the presence of that blessed power,
-which daily broke forth amongst us in our meetings,
+which daily broke forth among us in our meetings,
 through which we were greatly comforted, strengthened and edified;
 for it was that same Comforter our blessed Lord promised he would pray the Father for,
 and which the Father should send.
-John 14:16 and 26 This Comforter being come and received,
+John 14:16 and 16:26. This Comforter being come and received,
```
Comment (Henderjay) [inline journal-letters/modernized/03-chapter-1.adoc]: Here he actually means John 14: verses 16 and 26. But you were right to add the period after the 26.

File: journal-letters/modernized/03-chapter-1.adoc [hunk 3]
```diff
@@ -273,7 +273,7 @@ wherein the blessedness does consist.
 So then the Lord becoming our Shepherd, he taught us,
 and led us forth into green pastures,
 where we did feed and rest together with great delight.
-O! the joy, the pleasure, and the great delight,
+O--the joy, the pleasure, and the great delight,
```
Comment (Henderjay) [inline journal-letters/modernized/03-chapter-1.adoc]: Here again lets keep the exclamation point. `O!--the joy,`

File: journal-letters/modernized/03-chapter-1.adoc [hunk 4]
```diff
@@ -452,8 +452,8 @@ destitute and without a habitation.
 And then we saw our sacrificing and our sinning to be alike in the sight of God;
 for our prayers were rejected, and all loathed,
 because both were done in one nature and from one and the same seed and corrupt heart;
-and, therefore, it came to be with us as with Judah of old, as may be read Isa.
-1 and Isa. 46:3. where the Lord told Judah, their killing an ox,
+and, therefore, it came to be with us as with Judah of old, as may be read in
+Isa.1 and Isa. 66:3, where the Lord told Judah, their killing an ox,
```
Comment (Henderjay) [inline journal-letters/modernized/03-chapter-1.adoc]: This is right, but I think we just need a space between Isa. and 1 (like there is a space between Isa. and 63)

File: journal-letters/modernized/04-chapter-2.adoc [hunk 5]
```diff
@@ -37,12 +37,12 @@ and to dwell in that holy fear which the Father thereby placed in my heart.
...
 when he bid the kings and judges of the earth be wise and learned; and further said,
 "`serve the Lord with fear, and rejoice with trembling.`"
-O! the bowings of my soul! the pleasant dread that dwelt upon my spirit,
+O!--the bowings of my soul! the pleasant dread that dwelt upon my spirit,
 and the reverent tremblings that came over my heart, which filled it with living joy,
 as with marrow and fatness!
 Then could I say in my heart with David, I will wash my hands in innocency,
 and compass your altar, O Lord.
-O! the pleasant drawing near, and that not unprepared,
+O--the pleasant drawing near, and that not unprepared,
```
Comment (Henderjay) [inline journal-letters/modernized/04-chapter-2.adoc]: Same here... leave in the ! before the dashes (like you did a few lines up in line 40)

**Review by Henderjay — CHANGES_REQUESTED** [review]
Great work Julie. I left a few comments on a couple little things that need to be changed.  About the missing chunks in the beginning of the modernized version... I think it must have been because I edited that part for the _Truth in the Inward Parts_ book, and forgot that I had done it. :(  I'm not sure how many chunks are missing. If you think you can easily find them and put them back in, that would be great. If that sounds like a pain, we can put it on the list for another time. Thanks!!

**Comment by juliegugge** [comment]
ok... all done...

**Comment by Henderjay** [comment]
looks good Julie, Merged.

**Pull request merged**